website/sidebars.json

```diff
@@ -1,10 +1,116 @@
 {
   "docs": {
-    "Docusaurus": ["doc1"],
-    "First Category": ["doc2"],
-    "Second Category": ["doc3"]
-  },
-  "docs-other": {
-    "First Category": ["doc4", "doc5"]
+    "Getting started": [
+       "instalation/instalation"
+      ],
+    "Albums": [
+       "albums/getAlbum",
+       "albums/getAlbumTracks",
+       "albums/getSeveralAlbums"
+      ],
+    "Artists": [
+       "artists/getArtist",
+       "artists/getArtistAlbums",
+       "artists/getArtistRelatedArtists",
+       "artists/getArtistTopTracks",
+       "artists/getSeveralArtists"
+    ],
+    "Browse": [
+       "browse/getCategory",
+       "browse/getCategoryPlaylists",
+       "browse/getCategories",
+       "browse/getFeaturedPlaylists",
+       "browse/getNewReleases",
+       "browse/getRecommendations"
+    ],
+    "Driver": [
```

Comment:
I think this section could be in the end, because it's not related to Spotify, but internals of the SDK :thinking: